### PR TITLE
lzcat, lzma, unlzma, unxz, xzcat: add page

### PR DESCRIPTION
**Pull request by Unickorn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/lzcat.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# lzcat
+
+> This command is an alias of `xz --format=lzma --decompress --stdout`.
+
```
Comment (navarroaxel) [inline pages/common/lzcat.md:4]:
```suggestion
> More information: <https://manned.org/lzcat>.

```

File: pages/common/lzma.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# lzma
+
+> This command is an alias of `xz --format=lzma`.
+
```
Comment (navarroaxel) [inline pages/common/lzma.md:4]:
```suggestion
> More information: <https://manned.org/lzma>.

```

File: pages/common/unlzma.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# unlzma
+
+> This command is an alias of `xz --format=lzma --decompress`.
+
```
Comment (navarroaxel) [inline pages/common/unlzma.md:4]:
```suggestion
> More information: <https://manned.org/unlzma>.

```

File: pages/common/unxz.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# unxz
+
+> This command is an alias of `xz --decompress`.
+
```
Comment (navarroaxel) [inline pages/common/unxz.md:4]:
```suggestion
> More information: <https://manned.org/unxz>.

```

File: pages/common/xzcat.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# xzcat
+
+> This command is an alias of `xz --decompress --stdout`.
+
```
Comment (navarroaxel) [inline pages/common/xzcat.md:4]:
```suggestion
> More information: <https://manned.org/xzcat>.

```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**